packages/matter.js/src/cluster/definitions/WindowCoveringCluster.ts

```diff
@@ -0,0 +1,934 @@
+
+        /**
+         * The CurrentPositionTiltPercentage attribute identifies the actual position as a percentage from 0% to 100%
+         * with 1% default step. This attribute is equal to CurrentPositionTiltPercent100ths attribute divided by 100.
+         *
+         * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.5.12
+         */
+        currentPositionTiltPercentage: OptionalAttribute(
+            0x9,
+            TlvNullable(TlvPercent.bound({ min: 0, max: 100 })),
+            { scene: true, persistent: true, default: null }
+        )
+    },
+
+    commands: {
+        /**
+         * Upon receipt of this command, the server will adjust the window covering to the tilt percentage specified in
+         * the payload of this command.
+         *
+         * If the command includes TiltPercent100thsValue, then TargetPositionTiltPercent100ths attribute
+         *
+         * shall be set to TiltPercent100thsValue. Otherwise the TargetPositionTiltPercent100ths attribute shall be set
+         * to TiltPercentageValue * 100.
+         *
+         * If a client includes TiltPercent100thsValue in the command, the TiltPercentageValue shall be set to to
+         * TiltPercent100thsValue / 100, so a legacy server which only supports TiltPercentageValue (not
+         * TiltPercent100thsValue) has a value to set the target position.
+         *
+         * If the server does not support the Position Aware feature, then a zero percentage shall be treated as a
+         * UpOrOpen command and a non-zero percentage shall be treated as an DownOrClose command. If the device is only
+         * a tilt control device, then the command SHOULD be ignored and a UNSUPPORTED_COMMAND status SHOULD be
+         * returned.
+         *
+         * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.6.7
+         */
+        goToTiltPercentage: Command(0x8, TlvGoToTiltPercentageRequest, 0x8, TlvNoResponse)
+    }
+});
+
+/**
+ * A WindowCoveringCluster supports these elements if it supports feature Lift.
+ */
+export const LiftComponent = ClusterComponent({
+    attributes: {
+        /**
+         * The NumberOfActuationsLift attribute identifies the total number of lift/slide actuations applied to the
+         * Window Covering since the device was installed.
+         *
+         * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.5.6
+         */
+        numberOfActuationsLift: OptionalAttribute(0x5, TlvUInt16, { persistent: true, default: 0 }),
+
+        /**
+         * The TargetPositionLiftPercent100ths attribute identifies the position where the Window Covering Lift will go
+         * or is moving to as a percentage.
+         *
+         * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.5.13
+         */
+        targetPositionLiftPercent100Ths: OptionalAttribute(
+            0xb,
+            TlvNullable(TlvPercent100ths.bound({ min: 0, max: 10000 })),
```

Comment:
formally auto bound

Reply:
Added rules to omit bounds for % if they match the default bounds